.github/workflows/test.yml

```diff
@@ -78,3 +78,14 @@ jobs:
       run: |
         ./verify.sh cluster
 
+  # test if kernel module can be loaded, this is very os and instance specific, ignore if it fails
+  test_kernel_module:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: ./
+      with:
+        kernel_module_names: rapl,intel_rapl_common
```

Comment:
Shall the module list also include "intel_rapl_msr"?

Reply:
actually none of these modules can be loaded on the azure VM. So the test on github runner doesn't really verify much :D

Reply:
added